server/service/osquery_utils/queries.go

```diff
@@ -774,7 +774,11 @@ func generateSQLForAllExists(subqueries ...string) string {
 	// Generate EXISTS clause for each subquery
 	var conditions []string
 	for _, query := range subqueries {
-		condition := fmt.Sprintf("EXISTS (%s)", query)
+		// Remove semicolons from the query to ensure subqueries
+		// are not terminated early (Issue #19401)
+		sanitized := strings.ReplaceAll(query, ";", "")
```

Comment:
Why not just remove the last `;`? (Just in case there's a `;` in a string in the query e.g. `SELECT * FROM foo where bar = 'abc;bcd'`). Unlikely I know but better safe than sorry.